### PR TITLE
Add servertime command

### PR DESCRIPTION
**Pull request by AnaRitaTorres**
I made a former pull request with the same name however during code alterartions I made a mistake while rebasing, so here's the code that was in my last commit. Which had the following description: Added servertime chat command that shows the current server time.

### CODE REVIEW

**Comment by panpawn** [comment]
@AnaRitaTorres - for future reference, please do not close a PR if you mess it up.  It can almost always be fixed.  If you need help with this in the future, please do not hesitate to PM me personally on PS main.

**Comment by AnaRitaTorres** [comment]
Thank you for your availability and I'm sorry, I didn't know what I supposed to do.

**Comment by panpawn** [comment]
No problem!

File: chat-plugins/info.js [hunk 1]
```diff
@@ -1037,6 +1037,13 @@ exports.commands = {
 		this.sendReplyBox("Uptime: <b>" + uptimeText + "</b>");
 	},
 
+	'!servertime': true,
+	servertime: function (target, room, user) {
+		if (!this.runBroadcast()) return;
+		let servertime = new Date();
+		this.sendReplyBox(`Server Time: <b>${servertime.toLocaleString()}</b>`);
```
Comment (Zarel) [inline chat-plugins/info.js]: PS practically never uses title case. This should be `Server time:`

**Comment by Zarel** [comment]
Thanks!

**Pull request merged**